### PR TITLE
[TD-12458]<fix>taosBenchmark restfult full response

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

**Review by afwerar — APPROVED** [review]
LGTM

File: inc/demo.h [hunk 1]
```diff
@@ -146,7 +146,7 @@
 #define DEFAULT_INSERT_INTERVAL 0
 #define DEFAULT_QUERY_TIME 1
 #define DEFAULT_PREPARED_RAND 10000
-#define DEFAULT_REQ_PER_REQ 30000
+#define DEFAULT_REQ_PER_REQ 10000
```
Comment (sangshuduo) [inline inc/demo.h:149]: I worry this change would significantly decrease the insertion performance.

**Review by sangshuduo — CHANGES_REQUESTED** [review]
see my comments

**Review by sangshuduo — COMMENTED** [review]
LGTM

**Pull request merged**